Doc/library/dbm.rst

```diff
@@ -8,7 +8,16 @@
 
 --------------
 
-:mod:`dbm` is a generic interface to variants of the DBM database:
+A DBM database is a persistent, dictionary-like object that stores keys and values as :class:`bytes`.
+
+
+.. seealso::
+
+   Module :mod:`shelve`
+      Persistence module that can store keys and values of types other than bytes.
+
```

Comment:
I'd prefer if this could be moved to L31 instead (that is; just before the reference part starts).

Reply:
The rationale of the change as proposed is that, if the user needs shelves, mentioning the different implementations of a dbm database is a waste of their time. If you still think it adds value in that case, I'll change it.

Reply:
I understand, but I think we should also consider the visual impact. IMO, it is slightly visually jarring with the relatively large "seealso" box in the middle of the general description; I'd prefer it if the general module description came first, and then the yellow "seealso" box.

Moreover, I'm not sure visually skipping a few lines is _"a waste of time"_, since all of this fits in a screenful anyway.

<details>

<summary>In the middle of the description</summary>

<img width="837" alt="Screenshot 2024-11-04 at 10 15 27" src="https://github.com/user-attachments/assets/ac69ccce-39cb-4708-b8eb-c7231a89d4ed">

</details>

<details>

<summary>Below description</summary>

<img width="823" alt="Screenshot 2024-11-04 at 10 16 54" src="https://github.com/user-attachments/assets/33aab002-3261-47d8-af19-1f7ee54f8175">

</details>

Reply:
Alright, followed your suggestion.